src/components/Projects/ActionableLinksView.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useMemo, useState } from 'react'
+
+import { useMutation, useQueryClient } from '@tanstack/react-query'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { ZodSchema, z } from 'zod'
+
+import { Governance } from '../../clients/Governance'
+import useFormatMessage from '../../hooks/useFormatMessage'
+import { getProjectQueryKey } from '../../hooks/useProject.ts'
+import { Project, ProjectLink } from '../../types/proposals'
+import Link from '../Common/Typography/Link.tsx'
+import ErrorMessage from '../Error/ErrorMessage.tsx'
+import { BreakdownItem } from '../GrantRequest/BreakdownAccordion'
+import BlueLinkIcon from '../Icon/BlueLinkIcon.tsx'
+import Trashcan from '../Icon/Trashcan'
+
+import ActionableBreakdownContent from './ActionableBreakdownContent'
+import ExpandableBreakdownItem from './ExpandableBreakdownItem'
+import ProjectSidebarForm, { ProjectSidebarFormFields } from './ProjectSidebarForm'
+import ProjectSidebarSectionTitle from './ProjectSidebarSectionTitle'
+
+interface Props {
+  links: ProjectLink[]
+  projectId: string
+  isEditor: boolean
+}
+
+const NewLinkSchema: ZodSchema<Pick<ProjectLink, 'label' | 'url'>> = z.object({
+  label: z.string().min(1, 'Label is required').max(80),
+  url: z.string().min(0).max(200).url(),
```

Comment:
`min(0)` means optional?

Reply:
yes, there is an `optional` but it was not working with the `url` fn